### DISCUSSION TITLE
How to add custom Domain using fm?

### DISCUSSION THREAD

**Discussion by Venkateshvenki404224** [rtCamp/Frappe-Manager#227] [Q&A]
I have successfully developed and deployed a website using FM. I now want to move it to production and configure it to use a custom domain that I purchased through Hostinger. Could you provide guidance on the steps required to accomplish this?

**Comment by Xieyt** [comment]
Currently this specific use case is not supported as of now. Feature requests are always welcome.

For now, you can create new site with ssl for your domain and do the setup again.
